common/utils.py

```diff
@@ -142,17 +142,19 @@ def get_editable_obj_from_global_id(info, global_id, expected_obj_type):
             f"{obj.__class__.__name__} model does not implement is_editable_by_user()."
         )
     raise PermissionDenied(
-        f"User does not have permission to edit this {expected_obj_type.__name__}"
+        f"User does not have permission to edit this {expected_obj_type.__name__}."
     )
 
 
 def raise_permission_denied_if_not_staff(user):
```

Comment:
To make the staff vs. event staff distinction clearer shouldn't this function be renamed also?
```suggestion
def raise_permission_denied_if_not_event_staff(user):
```